Applications/Terminal/main.cpp

```diff
@@ -57,6 +57,7 @@
 static void run_command(int ptm_fd, String command)
 {
     pid_t pid = fork();
+
```

Comment:
I won't complain about this extra newline that has nothing to do with this commit if you add proper handling for the `pid < 0` case :smile:

Reply:
what if I just remove the extra newline :thinking: